### PR TITLE
enhance README

### PR DESCRIPTION
**Pull request by warm-coolguy**
## Summary

Extend README by plugin list and more detailed "getting started" instructions depending on user role.

## Instructions for local reproduction and review

View the file.

https://github.com/Dataport/polar/tree/enhance/README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,13 +14,48 @@ POLAR is built to serve both worlds. For generic use cases, generic clients are
...
+
+The most common use case for this client is in citizen's application processes regarding public service.
+
+Other clients with more specific code (and more visibility) include the [Denkmalkarte Schleswig-Holstein](https://efi2.schleswig-holstein.de/dish/dish_client/index.html), a memorial map, and the [Meldemichel Hamburg](https://static.hamburg.de/kartenclient/prod/), a map to inspect and create reports regarding damages to public infrastructure. The latter is currently being migrated to the version seen in this repository.
+
+## Plugins
+
+|Name|Details|
+|-|-|
+|[AddressSearch](https://github.com/Dataport/polar/tree/main/packages/plugins/AddressSearch)|Offers a search field and standard search service implementations with API for your own configurable custom search services. Integration with Reverse Geocoder and Pins possible, or usable as a data source for further processing.|
```
Comment (dopenguin) [inline README.md:27]:
```suggestion
|[AddressSearch](https://github.com/Dataport/polar/tree/main/packages/plugins/AddressSearch)|Offers a search field and standard search service implementations with API for your own configurable custom search services. For already usable search services, see the documentation of the package. Integration with Reverse Geocoder and Pins possible, or usable as a data source for further processing.|
```
Or something along those lines :D

File: README.md [hunk 2]
```diff
@@ -14,13 +14,48 @@ POLAR is built to serve both worlds. For generic use cases, generic clients are
...
+|[Scale](https://github.com/Dataport/polar/tree/main/packages/plugins/Scale)|Shows current scale as ratio and size indicator.|
+|[Toast](https://github.com/Dataport/polar/tree/main/packages/plugins/Toast)|Shows information to the user. Configurable in many plugins to communicate status updates or procedural advice.|
+|[Zoom](https://github.com/Dataport/polar/tree/main/packages/plugins/Zoom)|Allows zooming in and out of the client with buttons.|
+
 ## Getting started 🚀
 
-Clone the repository and run `npm install && npm run afm:dev`. This winds up our generic client in a minimal test environment. The most common use case for this client is in citizen's application processes regarding public service.
+### Using POLAR
+
+Select a client from our [releases](https://github.com/Dataport/polar/releases). For a generic client, use a `@polar/client-afm`. The zipped client is downloadable and attached as an asset to the release.
```
Comment (dopenguin) [inline README.md:48]:
```suggestion
Select a client from our [releases](https://github.com/Dataport/polar/releases). For a generic client, use `@polar/client-afm`. The zipped client is downloadable and attached as an asset to the release.
```

File: README.md [hunk 3]
```diff
@@ -14,13 +14,48 @@ POLAR is built to serve both worlds. For generic use cases, generic clients are
...
+
+The package contains example files that illustrate how the client can be used. You may also inspect the usage [here](https://github.com/Dataport/polar/tree/main/packages/clients/afm/example).
```
Comment (dopenguin) [inline README.md:50]:
```suggestion
The package contains example files that illustrate how the client can be used. You may also the example usage [here](https://github.com/Dataport/polar/tree/main/packages/clients/afm/example).
```

**Review by dopenguin — CHANGES_REQUESTED** [review]
Just some rough suggestions but looks really cool overall!

🏓 @warm-coolguy

File: README.md (hunk 3 above)
Comment (warm-coolguy) [reply README.md:50]: You may also the verb.

**Comment by warm-coolguy** [comment]
> 🏓 @warm-coolguy

🏓 @dopenguin quick

**Review by dopenguin — APPROVED** [review]
🚀 @warm-coolguy

**Pull request merged**